elements/pfe-tooltip/pfe-tooltip.ts

```diff
@@ -0,0 +1,104 @@
+ * @cssprop     {<length>} --pf-c-tooltip__content--PaddingLeft
+ *              Left Padding for the tooltip.
+ *              {@default `0.5rem`}
+ * @cssprop     --pf-c-tooltip__content--FontSize
+ *              Font size for the tooltip content.
+ *              {@default `0.875rem`}
+ * @cssprop     {<length>} --pf-c-tooltip__arrow--Width
+ *              Tooltip arrow width.
+ *              {@default `0.5rem`}
+ * @cssprop     {<length>} --pf-c-tooltip__arrow--Height
+ *              Tooltip arrow height.s
+ *              {@default `0.5rem`}
+ * @cssprop     --pf-c-tooltip__arrow--m-top--TranslateX
+ *              Positions the tooltip arrow along the x axis for `top` positioned arrows.
+ *              {@default `-50%`}
+ * @cssprop     --pf-c-tooltip__arrow--m-top--TranslateY
+ *              Positions the tooltip arrow along the y axis for `top` positioned arrows.
+ *              {@default `50%`}
+ * @cssprop     --pf-c-tooltip__arrow--m-top--Rotate
+ *              Rotates the tooltip arrow based on degrees of movement for `top` positioned arrows.
+ *              {@default `45deg`}
+ * @cssprop     --pf-c-tooltip__arrow--m-right--TranslateX
+ *              Positions the tooltip arrow along the x axis for `right` positioned arrows.
+ *              {@default `-50%`}
+ * @cssprop     --pf-c-tooltip__arrow--m-right--TranslateY
+ *              Positions the tooltip arrow along the y axis for `right` positioned arrows.
+ *              {@default `-50%`}
+ * @cssprop     --pf-c-tooltip__arrow--m-right--Rotate
+ *              Rotates the tooltip arrow based on degrees of movement for `right` positioned arrows.
+ *              {@default `45deg`}
+ * @cssprop     --pf-c-tooltip__arrow--m-bottom--TranslateX
+ *              Positions the tooltip arrow along the x axis for `bottom` positioned arrows.
+ *              {@default `-50%`}
+ * @cssprop     --pf-c-tooltip__arrow--m-bottom--TranslateY
+ *              Positions the tooltip arrow along the y axis for `bottom` positioned arrows.
+ *              {@default `-50%`}
+ * @cssprop     --pf-c-tooltip__arrow--m-bottom--Rotate
+ *              Rotates the tooltip arrow based on degrees of movement for `bottom` positioned arrows.
+ *              {@default `45deg`}
+ * @cssprop     --pf-c-tooltip__arrow--m-left--TranslateX
+ *              Positions the tooltip arrow along the x axis for `left` positioned arrows.
+ *              {@default `50%`}
+ * @cssprop     --pf-c-tooltip__arrow--m-left--TranslateY
+ *              Positions the tooltip arrow along the y axis for `left` positioned arrows.
+ *              {@default `-50%`}
+ * @cssprop     --pf-c-tooltip__arrow--m-left--Rotate
+ *              Rotates the tooltip arrow based on degrees of movement for `left` positioned arrows.
+ *              {@default `45deg`}
+ */
+@customElement('pfe-tooltip')
+export class PfeTooltip extends BaseTooltip {
+  static readonly version = '{{version}}';
+
+  static readonly styles = [style];
```

Comment:
I think we ought to apply these styles in `BaseTooltip` WDYT? in #2042 i ended up splitting the stylesheet in half, one common to pfe and rhds for the base, and one specific to pfe for the subclass

Reply:
I think that makes sense.  I don't know that we'll have any specific to pfe at this point but I do think that having base styles will be important for maintainability between rhds & pfe

Reply:
i noticed there's still a pfe-tooltip.scss but it's not imported or applied in PfeTooltip